src/metrics/aggregation/type.go

```diff
@@ -249,8 +249,23 @@ func validateProtoType(a aggregationpb.AggregationType) error {
 
 // ParseType parses an aggregation type.
 func ParseType(str string) (Type, error) {
-	aggType, ok := typeStringMap[str]
-	if !ok {
+	var (
+		aggType    Type
+		exactMatch bool
+		looseMatch bool
+	)
+
+	aggType, exactMatch = typeStringMap[str]
+	if !exactMatch {
+		for key, val := range typeStringMap {
+			if strings.ToLower(key) == strings.ToLower(str) {
+				looseMatch = true
+				aggType = val
```

Comment:
Do you want to `break` here?  I guess it doesn't matter too much.

Reply:
good call